docs/_posts/2016-09-21-this-week-in-react-1.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "This Week In React #1"
+author: ericnakagawa
+---
+
+_This Week In React is a weekly publication highlighting interesting projects and active members from the React community. [Sign up to the weekly newsletter.](http://eepurl.com/cgOjbX)_
+
+![Projects aimed at beginning React developers.](/react/img/blog/thisweekinreact/beginner.png)
+
+####📓 A Visual Intro to Redux & Relay####
+
+Lin Clark has developed a series of visual guides for learning Flux, Redux, and most recently released a four-part visual guide to Relay.
+
+
+Redux: [https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6](https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6)
```

Comment:
This is great feedback. I considered switching to text links. Thanks @zpao and @acdlite